src/components/atoms/BadgePreview/mini/MiniBadgePreview.tsx

```diff
@@ -17,19 +18,20 @@ const defaultValuesForMiniBadgePreviewProps = {
   animationEffects: ['wobble', 'grow'] as BadgePreviewEffects[],
 }
 
-const MiniBadgePreviewBox = styled(Box)<{ highlightColor?: string }>(({ highlightColor }) => ({
+const MiniBadgePreviewBox = styled(Box)<{ highlightColor?: string }>(() => ({
   width: 180,
-  height: 250,
-  border: `2px solid ${highlightColor || colors.white}`,
+  maxHeight: 300,
   borderRadius: '8px',
   display: 'flex',
   flexDirection: 'column',
   justifyContent: 'center',
   alignItems: 'center',
-  padding: '0 12px',
 }))
 
 export const MiniBadgePreview = (props: MiniBadgePreviewProps = defaultValuesForMiniBadgePreviewProps) => {
+  const theme = useTheme()
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const palette = theme.palette as any
```

Comment:
Why do you need to do this? Can you use `theme.palette` directly without casting it to any?